frontend/src/features/properties/hooks/useParcelLayerData.tsx

```diff
@@ -152,28 +155,55 @@ const useParcelLayerData = ({
   nameSpace,
   agencyId,
 }: IUseParcelLayerDataProps) => {
+  // Get LTSA information
+  const [ltsa, setLtsa] = useState<ILTSAOrderModel | undefined>(undefined);
+  const api = useApi();
+  // make the call to the LTSA api
+  const getLTSAData = async (pid: string) => {
+    try {
+      const ltsaData = await api.getLTSA(pid);
+      setLtsa(ltsaData);
+      return ltsaData;
+    } catch (e) {
+      setLtsa(undefined);
+    }
+  };
   const parcelLayerData = useAppSelector((store) => store.parcelLayerData?.parcelLayerData);
   const { getByType } = useCodeLookups();
   const [showOverwriteDialog, setShowOverwriteDialog] = useState(false);
   const dispatch = useAppDispatch();
   const keycloak = useKeycloakWrapper();
 
   useDeepCompareEffect(() => {
-    if (!!formikRef?.current && isMouseEventRecent(parcelLayerData?.e) && !!parcelLayerData?.data) {
-      if (!parcelId) {
-        setParcelFieldsFromLayerData(
-          parcelLayerData,
-          formikRef,
-          getByType(AMINISTRATIVE_AREA_CODE_SET_NAME),
-          nameSpace ?? '',
-          agencyId ?? keycloak.agencyId,
-        );
-        dispatch(clearParcelLayerData());
-      } else {
-        setShowOverwriteDialog(true);
+    // Define an async function to fetch LTSA data
+    const fetchLTSAData = async () => {
+      if (
+        !!formikRef?.current &&
+        isMouseEventRecent(parcelLayerData?.e) &&
+        !!parcelLayerData?.data
+      ) {
+        if (!parcelId) {
+          const pid = parcelLayerData?.data?.PID || parcelLayerData?.data?.PID_NUMBER?.toString();
+          const ltsaResponseData = await getLTSAData(pid);
```

Comment:
Then we would only need...
```suggestion
          const ltsaResponseData = await api.getLTSA(pid);
```